src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityLink.tsx

```diff
@@ -38,7 +55,7 @@ const AnnotationActivityLink = ({
     };
 
     const handleMouseDown = (event: React.MouseEvent<HTMLButtonElement>) => {
-        if (isDisabled) {
+        if (isFileVersionUnavailable) {
```

Comment:
Can we do something like `!fileVersion`? Or do we have to specifically check for `null`?